libraries/AP_Scripting/drivers/LTE_modem.lua

```diff
@@ -94,11 +96,11 @@ local LTE_SERVER_PORT = bind_add_param('SERVER_PORT',  8, 0)
 --[[
     // @Param: LTE_BAUD
     // @DisplayName: Serial Baud Rate
-    // @Description: Baud rate for the serial port to the LTE modem. If using something other than 115200 you need to connect to the modem and use AT+IPREX=BAUD to set the baud rate and save with AT&W. The fastest supported baudrate is 3686400.
+    // @Description: Baud rate for the serial port to the LTE modem when connected. Initial power on baudrate is in LTE_IBAUD
     // @Values: 19200:19200,38400:38400,57600:57600,115200:115200,230400:230400,460800:460800,921600:921600,3686400:3686400
     // @User: Standard
 --]]
-local LTE_BAUD        = bind_add_param('BAUD',  9, 115200)
+local LTE_BAUD        = bind_add_param('BAUD',  9, 921600)
```

Comment:
Why not pull from the serial baud rate params for the scripting port. (we really should make this easier.)